packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -51,6 +51,23 @@ type TApplicationContextDataFences = Partial<
   >
 >;
 type TApplicationContextEnvironment = ApplicationWindow['app'];
+type TApplicationContextUser = Pick<
+  NonNullable<TFetchedUser>,
+  'id' | 'email' | 'firstName' | 'lastName' | 'businessRole' | 'projects'
+> & {
+  locale: string;
+  timeZone: string;
+  idTokenUserInfo?: {
```

Comment:
Nit: we could also use the generated type `TIdTokenUserInfo` from `generated/mc.ts`. The only thing to change is the type of `additionalClaims` but I think it could be done using `Omit` and `&`.

Reply:
Updated in this commit: 0138b05933457a87daefeace91129884567af75a